ansibleflow/app.py

```diff
@@ -50,13 +50,29 @@ def setup_argument_parser():
     )
 
     run_parser = subparsers.add_parser('run')
-    run_parser.add_argument('--env', type=str, default='default')
     run_parser.add_argument(
         'run_action',
-        nargs='*',
+        type=str,
         default=True,
         help=argparse.SUPPRESS
     )
+    run_parser.add_argument(
+            '--env',
+            type=str,
+            default='default')
+    run_parser.add_argument(
+            "-v",
+            "--verbosity",
+            action="count",
+            default=0,
+            help="increase output verbosity")
```

Comment:
So, the way that I've dealt with this before was to just to use the ansible-options value on a given target. This allows for you to specify any number of custom options you want to be passed on to ansible.